pkgs/development/python-modules/inform/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildPythonPackage, fetchPypi
+, arrow
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "inform";
+  version = "1.23.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0dvc5klbnbryrvspp45nmlg02g40j7xspcz7lqsm0c0dj0z29zdz";
+  };
+
+  propagatedBuildInputs = [ arrow six ];
+
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

Reply:
Is this solved?

Reply:
Resolved. There are apparently no tests in the copy on PyPI, but the source on GitHub comes with a lot of tests, which are now running and passing.